billboard.py

```diff
@@ -226,6 +228,18 @@ def fetchEntries(self):
                 message = "Failed to parse title"
                 raise BillboardParseException(message)
 
+            try:
+                coverSoup = entrySoup.find('img',"chart-list-item__image")
```

Comment:
I think this ought to reuse `_ROW_SELECTOR_FORMAT` in some way.

Reply:
Unable to re use _ROW_SELECTOR_FORMAT as it uses div.